pkg/agent/openflow/client.go

```diff
@@ -631,6 +679,9 @@ func (c *client) InstallGatewayFlows() error {
 	if gatewayConfig.IPv4 != nil {
 		gatewayIPs = append(gatewayIPs, gatewayConfig.IPv4)
 		flows = append(flows, c.gatewayARPSpoofGuardFlow(gatewayConfig.IPv4, gatewayConfig.MAC, cookie.Default))
+		// This flow is used to allow the response Service packet whose Endpoint is on host network to get ARP resolution.
+		// The source IP of the ARP is Node IP, not Antrea gateway's IP.
```

Comment:
Do you mean if the Node has multiple IPs, and the Node IP is not the default route, the ARP request can use a different IP and be dropped?

Reply:
I will delete the flow as I tested NodePort, LoadBalancer, ClusterIP e2e cases, and no packet hits the flow.